test/1_stdlib/Character.swift

```diff
@@ -222,7 +222,7 @@ CharacterTests.test("RoundTripping/Random") {
 CharacterTests.test("forall x: ASCII . String(Character(x)) == String(x)") {
   // For all ASCII chars, constructing a Character then a String should be the
   // same as constructing a String directly.
-  let asciiDomain = Array(0..<128).map({ UnicodeScalar(Int($0)) })
+  let asciiDomain = Array(0..<128).map({ UnicodeScalar(Int($0))! })
```

Comment:
While you are working with this code, you can simplify it by dropping an unnecessary temporary array:

```swift
let asciiDomain = (0..<128).map({ UnicodeScalar(Int($0))! })
```